homeassistant/components/cloud/tts.py

```diff
@@ -277,7 +277,7 @@ def handle_deprecated_gender(
         hass,
         DOMAIN,
         "deprecated_gender",
-        is_fixable=True,
+        is_fixable=False,
```

Comment:
These she's till be fixable. We should return a default fxi flow for these.

Reply:
How should they be fixable?
The user need to manually adjust things?

Reply:
For service calls we create a fixable repair issue that the user can confirm when they have updated their automations and scripts.